therapy/etl/hemonc.py

```diff
@@ -190,22 +190,22 @@ def _get_concepts(self) -> Tuple[Dict, Dict, Dict]:
         return therapies, brand_names, conditions
 
     @staticmethod
-    def _id_to_yr(hemonc_id: str) -> int:
+    def _id_to_yr(hemonc_id: str) -> str:
         """Get year from HemOnc ID corresponding to year concept.
         :param str hemonc_id: HemOnc ID to get year for
-        :return: int representing year. Raises TypeError if HemOnc ID not valid.
+        :return: str representing year. Raises TypeError if HemOnc ID not valid.
         """
         id_int = int(hemonc_id)
         if id_int == 780:
-            return 9999
+            return "9999"
         elif id_int == 48349:
-            return 2020
+            return "2020"
         elif id_int == 5963:
-            return 2021
+            return "2021"
         elif id_int < 699 or id_int > 780:
             raise TypeError("ID not a valid HemOnc year concept")
         else:
-            return id_int + 1240
+            return str(id_int + 1240)
```

Comment:
Should update line 252 to be `if year == "9999"`, should also update schemas from int --> str

Reply:
oof good catch